test/built-ins/Proxy/getOwnPropertyDescriptor/resultdesc-is-not-configurable-not-writable-targetdesc-is-writable.js

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2019 Aleksey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.5
+description: >
+    Throws a TypeError exception if resultDesc is both non-configurable and
+    non-writable, while targetDesc is writable.
+info: |
+    [[GetOwnProperty]] (P)
+
+    ...
+    17. If resultDesc.[[Configurable]] is false, then
+        ...
+        b. If resultDesc has a [[Writable]] field and resultDesc.[[Writable]] is
+        false, then
+            i. If targetDesc.[[Writable]] is true, throw a TypeError exception.
+    ...
+features: [Proxy]
+---*/
+
+var target = {};
+var p = new Proxy(target, {
+  getOwnPropertyDescriptor: function(t, prop) {
+    return {
+      configurable: false,
+      writable: false,
+    }
+  },
+});
+
+Object.defineProperty(target, "prop", {
+  configurable: false,
+  writable: true,
+});
```

Comment:
like the other example above, please add another test that would add the property inside the proxy trap.

Reply:
17ed315